### ISSUE TITLE
0.1.11 Breaks Chrome launching on OS X

### ISSUE THREAD

**Issue by joechrz** [karma-runner/karma-chrome-launcher#49]
Looks like this was introduced by d442548e3ca91bb200ddeca9d1384155bc499056

From what I can tell `fs.realpathSync` throws an error if the path isn't found, but the code expects it to return true/false.  The error prevents the `defaultPath` param from getting registered and Chrome doesn't launch.

```
WARN [plugin]: Error during loading "/Users/joechrz/projects/fire-engine/node_modules/karma-chrome-launcher" plugin:
  ENOENT, no such file or directory '/Users/joechrz/Applications/Google Chrome.app'
INFO [framework.browserify]: registering rebuild (autoWatch=true)
[17:10:37] Starting 'serve'...
fire-engine.js was browserified: 0.701s
WARN [karma]: Port 9876 in use
[17:10:37] Finished 'serve' after 1.5 ms
INFO [karma]: Karma v0.12.31 server started at http://localhost:9877/
WARN [launcher]: Can not load "Chrome", it is not registered!
  Perhaps you are missing some plugin?
```

**Comment by JeffreyMcBride** [comment]
I'm getting the same behavior on Windows 7.  v0.1.8 worked fine with and without chrome_bin set.  v0.1.11 fails to find chrome.exe either way.

**Comment by robmcguinness** [comment]
I had to roll back to [v0.1.10](https://github.com/karma-runner/karma-chrome-launcher/releases/tag/v0.1.10) for Chrome to launch properly on OSX

**Comment by alias-mac** [comment]
@JeffreyMcBride, can you check if #52 is working for windows on all the prefixes?
Thanks and sorry for causing this bug.

**Comment by zzo** [comment]
fix published to npm  v0.1.12

**Comment by JeffreyMcBride** [comment]
Thanks, yes it was fixed with all prefixes and v0.1.12 is working fine...
